cli/crates/typed-resolvers/src/codegen.rs

```diff
@@ -188,12 +188,6 @@ fn write_resolver_type<O: fmt::Write>(schema: &AnalyzedSchema<'_>, out: &mut O)
     writeln!(out, "}}\n")
 }
 
-const RESOLVER_FN_DEFINITION: &str = r#"
-import * as sdk from '@grafbase/sdk'
-type ResolverFn<Parent, Args, Return> = ((parent: Parent, args: Args, context: sdk.Context, info: sdk.Info) => Return) |
```

Comment:
Wouldn't you still need the latter part, even if you're importing in the client?

Reply:
Or is this duplicated elsewhere?

Reply:
Do you mean the `ResolverFn` type definition? It's exported from sdk now. But that made me realize there's a problem: we don't refer to it as `sdk.ResolverFn`. I just pushed a fix for that.